kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver_with_constraints.h

```diff
@@ -198,7 +198,7 @@ class ResidualBasedBlockBuilderAndSolverWithConstraints
         for (int k = 0; k < n_constraints; k++)
         {
             auto it = constraints_begin + k;
-            it->InitializeSolutionStep(); // Here each constraint constructs and stores its T and C matrices. Also its equation ids.
+            it->InitializeSolutionStep(rModelPart.GetProcessInfo()); // Here each constraint constructs and stores its T and C matrices. Also its equation ids.
```

Comment:
Better to move this outside the loop, you are accessing to the process info everytime